pkg/util/kudo/convert_types.go

```diff
@@ -13,3 +17,25 @@ func StringValue(v *string) string {
 	}
 	return ""
 }
+
+// YAMLList converts YAML input describing a list.
+func YAMLList(v string) ([]interface{}, error) {
```

Comment:
should be moved

Reply:
IMO, `convert_types.go` is the right place for this function. The package needs a different name though.